### PR TITLE
[MXNET-782] Fix Custom Metric Creation in R tutorial

### PR DESCRIPTION
**Pull request by anirudhacharya**
## Description ##
Fix custom metric creation in tutorial

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

## Comments ##
@hetong007 @ankkhedia

### CODE REVIEW

**Comment by hetong007** [comment]
Code to test if a package is installed, if not then install it:

```r
if (!require(mlbench)) {
    install.packages('mlbench')
}
```

File: docs/tutorials/r/fiveMinutesNeuralNetwork.md [hunk 1]
```diff
@@ -1,27 +1,31 @@
...
 
  ```
-    ## Loading required package: mlbench
+    ## Loading required package: mxnet
  ```
 
  ```r
-    require(mxnet)
+    if (!require(mxnet)) {
+      install.packages('mlbench')
```
Comment (sandeep-krishnamurthy) [inline docs/tutorials/r/fiveMinutesNeuralNetwork.md]: Why test on mxnet and install mlbench?

Comment (anirudhacharya) [reply docs/tutorials/r/fiveMinutesNeuralNetwork.md]: this was a typo, i will fix it

**Comment by ankkhedia** [comment]
Looks good!

**Comment by sandeep-krishnamurthy** [comment]
@hetong007 - Thanks for your review. I see your comments are addressed. Merging.

**Pull request merged**